### ISSUE TITLE
Add a way to open the more-info panel for the temperature entity

### ISSUE THREAD

**Issue by beauwest** [Makin-Things/platinum-weather-card#87]
**Describe the solution you'd like**
When looking at temperature, I really like to look at the trends as well. I'd love a way to be able to click or tap on the current temperature number and have it open up the more-info card for the entity where that number pulls from.

![Screenshot 2023-04-27 at 12 46 09 PM](https://user-images.githubusercontent.com/193887/234948908-9bd76772-caa0-4191-ac6b-ec5c7cf5c9e9.png)

**Comment by JOHLC** [comment]
I would like to be able to click on any of the entities and see the more-info dialog

**Comment by kineticscreen** [comment]
Agreed, would be great if there was an option to enable this for all of the slots.

**Comment by JDuinkerken** [comment]
Fully agreed!
